sql/executor.go

```diff
@@ -134,7 +136,10 @@ func runExec(slct string, db *DB) *PipeReader {
 
 		err := func() error {
 			startAt := time.Now()
-			log.Infof("Starting runStanrardSQL1:%s", slct)
+			log.Infof("Starting runExec:%s", slct)
```

Comment:
Maybe we only need time elapsed for the `slct` statement in the log, so this line can be `Debugf`? And below elapsed log can log together with `slct`, or it maybe harder to figure out which statement this time was.

Reply:
Sounds good. :)